backend/internal/controllers/book_test.go

```diff
@@ -51,8 +77,8 @@ func TestGetBooks(t *testing.T) {
 func TestGetBookDetailsByID(t *testing.T) {
 
 	var response models.BookDetails
-	testutils.MakeHttpRequest("GET", ts.URL+"/books/catcher", &response, t)
+	testutils.MakeHttpRequest("GET", ts.URL+"/books/catcher/en", &response, t)
 
 	// Check title
-	require.Equal(t, "catcher in the rye", response.Title)
+	require.Equal(t, "catcher_rb", response.Read.Body)
```

Comment:
Can we add the title check back in too please? And maybe some more tests for:

* Book not found
* Book found, but no contents for that book in the specified lang
* Test with a book that has contents in multiple languages, just to double-check we're selecting the correct language